ansible/db-backup/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
     path: ~/backups
     state: directory
 - name: fetch (or update) the container
-  command: docker pull ghcr.io/infinite-industries/ops:latest
+  command: docker pull ghcr.io/infinite-industries/ops-server:{{ image_version }}
```

Comment:
ok now i see that ops-server was likely just a rename from the image formerly called ops, but what was ops where where/how is it built and published?